docs/UserGuide.adoc

```diff
@@ -18,12 +18,12 @@ By: `W09-4`      Since: `Feb 2019`      Licence: `MIT`
 
 == Introduction
 
-Quickdocs is a program that is catered for doctors practicing in small family clinics in Singapore. It integrates the patient, appointment and stock management into a single application. Doctors can better manage their clinics while still maintaining a small manpower size.
+QuickDocs is an all in one solution where doctors can have greater control in facilitating patient consultations, organizing appointments, and monitoring financial and inventory records in private clinics.
```

Comment:
Perhaps we can use something like this:

QuickDocs is an all in one solution granting doctors with greater control in their private clinics by facilitating patient consultations, organizing appointments and monitoring financial and inventory records.